### ISSUE TITLE
Edit and update math function not working

### ISSUE THREAD

**Issue by binnyjames** [MO-Movia/licit#26]
Insert a math function. Select the inserted math function. Click on Edit button. In math editor, edit the text and click on 'Update' button.
Result: The edited text is not updated in editor.

**Comment by tedchou12** [comment]
this one, I tested, the edit seems to be working correctly. Anyone replicated the issue?

**Comment by MO-Movia** [comment]
Yes, I duplicated in Chrome.  However, I did a refresh, and it started working.  Hmmmmm. Wonder if there was an update that was cached?

**Comment by MO-Movia** [comment]
@binnyjames, can you check again, and list which browser you are using when you see the error?  Thanks!

**Comment by tedchou12** [comment]
Sorry, this does exist.

**Comment by MO-Movia** [comment]
So, @tedchou12, you were able to duplicate?  Which browser? And it appears to work sometimes, and not others?

**Comment by tedchou12** [comment]
yes. Chrome.

Seems like is not as stable.


If you try to highlight the formula and edit, then it works, if no highlight, it will not work (the usual way).

I took a look at the code, is beyond my ability 😅

**Comment by MO-Movia** [comment]
Strange, now I can't duplicate it no matter what I try on greathints demo. But it did it the first time I tried it before I refreshed. Guess we will get the experts at Firmusoft to take a look and see if they can find it.  But it would be better if we could find a specific pattern that causes it to happen. Doing it some of the time is hard to track down.

**Comment by binnyjames** [comment]
Steps tried (see image):
![math_editor_update_issue](https://user-images.githubusercontent.com/72541915/95556919-ac942f80-0a31-11eb-8909-6222d8cc0b33.gif)

1. Select a math function
2. Click on math function's 'Edit' helper icon
3. In Math Editor, type '+4'
4. In Math Editor, click on the Update button. Result: Math function not showing the edited '+4'
5. Repeat steps 1 and 2.  In Math Editor the edited text is missing

Browser: Chrome
OS: Windows 10
